### DISCUSSION TITLE
Relation of Youtube app and Pipepipe user data

### DISCUSSION THREAD

**Discussion by kayprish** [InfinityLoop1308/PipePipe#631] [Q&A] [answered]
I don't know enough about how the YouTube recommendation system works, but from my understanding if a user doesn't have an account, YouTube attempts to identify the user based on certain identification, using his device but also his cookies, and makes recs based on this identity.
Clearing your browser history is usually enough for your recommendations to temporarily revert to some non-user-tailored default in your region of the world.
My question is whether, while using both PipePipe and the YouTube app for watching different types of content, or for content in different languages respectively. Will my recommendations in both apps tend to converge or not?
For example, if I only watch English content on PipePipe, and only German content on the Youtube app, will I get recs in different languages, as I'd expect, or not?

**Comment by InfinityLoop1308** [comment, marked as answer]
PipePipe doesn't use cookies or share data with YouTube, unlike the official YouTube app which uses cookies and other tracking methods to personalize recommendations. Since PipePipe maintains privacy by not using these tracking features, your viewing habits on PipePipe won't affect your YouTube app recommendations and vice versa. So in your example, watching English content on PipePipe and German content on YouTube app will keep the recommendations separate - you'll continue getting English suggestions on PipePipe based on trending/popular videos, while YouTube app will recommend German content based on your watching history there.
